kit/platform/id.go

```diff
@@ -143,3 +144,20 @@ func (i ID) MarshalText() ([]byte, error) {
 func (i *ID) UnmarshalText(b []byte) error {
 	return i.Decode(b)
 }
+
+// Value implements the database/sql Valuer interface for adding IDs to a sql database.
+func (i ID) Value() (driver.Value, error) {
+	return i.String(), nil
+}
+
+// Scan implements the database/sql Scanner interface for retrieving IDs from a sql database.
+func (i *ID) Scan(value interface{}) error {
+	switch v := value.(type) {
+	case int64:
+		return i.DecodeFromString(strconv.Itoa(int(v)))
```

Comment:
Why the cast to `int` here when `v` is `int64` and `itoa` is 64-bit?

Reply:
I wasn't able to get it to work with `Itoa` since that function needs a regular `int`, but I was able to get it to work with `FormatInt` which takes an `int64` - the performance seems just about identical, but using `FormatInt` seems a little cleaner.